website/docs/reference/developer-readme.md

```diff
@@ -35,6 +61,8 @@ description: SynapseML Development Setup
     `horovod` requirement in the environment.yml file, because horovod installation only
     supports Linux or macOS. Horovod is used only for namespace `synapse.ml.dl`.
     :::
+1. On Windows, install WinUtils
+    - Download [WinUtils.exe](https://github.com/steveloughran/winutils/blob/master/hadoop-3.0.0/bin/winutils.exe) and copy it into the `bin` directory of your Spark installation, e.g. C:\Users\user\AppData\Local\Spark\spark-3.3.2-bin-hadoop3\bin
```

Comment:
Steps should be
Download Winutils
Place it in C:\Program Files\Hadoop\bin creating folders if needed
Edit the environment variables to set HADOOP_HOME to C:\Program Files\Hadoop
Edit the path to contain %HADOOP_HOME%\bin